client/src/app/components/filters/filters.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+import { StateService } from 'src/app/services/state.service';
+
+@Component({
+  selector: 'app-filters',
+  templateUrl: './filters.component.html',
+  styleUrls: ['./filters.component.scss'],
+})
+export class FiltersComponent implements OnInit {
+  public topics = ['people', 'cash', 'data', 'all'];
```

Comment:
There is an enum-file with these hard-coded values somewhere...
See: https://github.com/rodekruis/cash-program-design-wizard/blob/main/client/src/app/models/tag.enum.ts

And I'd stick with the "tag" name for everything, so no need to introduce a different term/concept like "topic", I think.